app/scripts/modules/core/src/help/HelpMenu.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import { Dropdown, MenuItem, Glyphicon } from 'react-bootstrap';
+
+import './HelpMenu.less';
+
+export const HelpMenu = () => {
+  return (
+    <li className="help-menu">
+      <Dropdown id="help-menu-dropdown" pullRight={true}>
+        <Dropdown.Toggle className="hidden-lg" noCaret={true}>
+          <Glyphicon glyph="question-sign" />
+        </Dropdown.Toggle>
+        {HelpMenuItems(true)}
```

Comment:
Should be `<HelpMenuItems divide={true} />`

Reply:
I think it might be a little confusing to have the divider in some cases and not others, but I don't feel strongly about it.

Reply:
re: the divider, agreed there's not a lot of benefit from it but at the moment I'm just matching style with the existing auth menu in mobile view:

![auth_menu](https://user-images.githubusercontent.com/34253460/36604957-4119913e-188d-11e8-8808-05961a1e5e44.png)

I can remove the divider completely from both menus in a follow-up PR.

re: `<HelpMenuItems divide={true}>`, react-bootstrap's `Dropdown` component isn't flexible enough to allow `DropdownMenu` to be encapsulated within another component.  It throws a couple of errors when done so.  Also after adding a typescript interface for { divide: boolean } and a default props object in the function body I ended up with about the same number of loc.  So I've DRY-ed the part of the code that matters, the link URLs, and the rest is duplicated for both menus.